src/test/standardTest.ts

```diff
@@ -92,6 +92,8 @@ async function createSettings(): Promise<string> {
         'python.insidersChannel': 'off',
         'jupyter.logging.level': 'debug',
         'python.logging.level': 'debug',
+        'python.experiments.enabled': 'true',
+        'python.experiments.optOutFrom': '[]',
```

Comment:
Isnt' this supposed to be a regular array
```suggestion
        'python.experiments.optOutFrom': [],
```

Reply:
Maybe that was causing the extension crash. Thanks.